src/api/app/controllers/webui/project_controller.rb

```diff
@@ -24,6 +24,8 @@ class Webui::ProjectController < Webui::WebuiController
   before_filter :load_releasetargets, :only => [ :show, :incident_request_dialog, :release_repository_dialog ]
   prepend_before_filter :lockout_spiders, :only => [:requests, :rebuild_time, :buildresults, :maintenance_incidents]
 
+  after_action :verify_authorized, :only => :save_new
```

Comment:
This will only ensure that we use the authorization in the action, otherwise it'll raise an exception!
And yes, only for the save_new action because this is the only action which uses the Pundit authorization atm.
https://github.com/elabs/pundit#ensuring-policies-are-used